public/app/features/plugins/admin/components/InstallControls/InstallControlsButton.tsx

```diff
@@ -164,6 +165,22 @@ export function InstallControlsButton({
     );
   }
 
+  const isOpenSource = config.buildInfo.edition === GrafanaEdition.OpenSource;
```

Comment:
[nitpick] maybe we should expose `isOpenSource` and `isEnterprise` etc as helper functions directly from config?